src/stores/settings.js

```diff
@@ -35,13 +35,24 @@ export const useSettingsStore = defineStore('settings', {
       const timeStr = `start=${this.startDate.toISOString()}&end=${this.endDate.toISOString()}`
       option = option ? `${option}&${timeStr}` : timeStr
       option += '&proposal_id=' + proposal
+      option += '&include_thumbnails=true'
       const imageUrl = option ? `${baseUrl}?${option}` : baseUrl
       const responseData = await fetchApiCall({ url: imageUrl, method: 'GET' })
       if (responseData && responseData.results) {
         // Preload all the small thumbnails into the cache. The large thumbnails will be loaded on demand
```

Comment:
does this comment need changing since it seems to be pre-loading the large thumbnails as well

Reply:
no its not preloading large thumbnails, its just storing the large and small thumbnail urls into the object if they exist in the response. But its only preloading the small ones into the cache.